lib/carrierwave/orm/activerecord.rb

```diff
@@ -81,7 +81,7 @@ def remove_#{column}!
         end
 
         # Reset cached mounter on record reload
-        def reload
+        def reload(*args)
```

Comment:
I see. And `DEPRECATION: `expect { }.not_to raise_error(SpecificErrorClass)` is deprecated. Use `expect { }.not_to raise_error` (with no args) instead`. So I will fix this DEPRECATION :)